crates/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -228,22 +227,23 @@ where
     /// uninstalled rather than a bool.
     async fn uninstall(
         &mut self,
-        packages: Vec<FloxPackage>,
-        nix: &NixCommandLine,
-        system: System,
-    ) -> Result<bool, EnvironmentError2> {
-        let current_manifest_contents = self.manifest_content()?;
-
-        let new_manifest_contents =
-            flox_nix_content_with_packages_removed(&current_manifest_contents, packages)?;
-        match new_manifest_contents {
-            ManifestContent::Unchanged => return Ok(false),
-            ManifestContent::Changed(contents) => {
-                self.transact_with_manifest_contents(contents, nix, system)
-                    .await?;
-                Ok(true)
-            },
-        }
+        _packages: Vec<FloxPackage>,
+        _nix: &NixCommandLine,
+        _system: System,
+    ) -> Result<Option<String>, EnvironmentError2> {
+        // let current_manifest_contents = self.manifest_content()?;
```

Comment:
Do we need this comments out code?

Reply:
No, it's being replaced in the `flox uninstall` PR that I'm about to put up. It's just easier to comment that out since the followup is so quick.